### DISCUSSION TITLE
[5.2.0] Maps on image detail is not updated when switching left right.

### DISCUSSION THREAD

**Discussion by Colossus68** [LycheeOrg/Lychee#2406] [Bugs] [answered]
Hi, I'm new to Lychee. 
Maps in the detail block of a photo are not updated when moving directly from one image to the next (swipe, keyboard or next/previous button). All other image data is updated, including GPS values. Only when I return to the album first and then open the next image is the map also updated. I have tested all map providers. Is this behaviour intentional?

**Comment by ildyria** [comment]
No they should be updated.
That used to be the case, it seems something is going wrong.

**Reply by Colossus68** [reply]
The JS-functions displayMap() and displayOnMap() are not called when the image is changed. They are only triggered when the image is called up from the album.

**Reply by ildyria** [reply, marked as answer]
Thanks, fix is on its way. :)

https://github.com/LycheeOrg/Lychee/pull/2423/files

**Reply by Colossus68** [reply]
Works like a charm! Thank you!
